src/pystack/_pystack/version.h

```diff
@@ -138,13 +125,15 @@ struct py_thread_v
 
 struct py_runtime_v
 {
-    typedef PyRuntimeState Structure;
     ssize_t size;
     FieldOffset<remote_addr_t> o_finalizing;
     FieldOffset<remote_addr_t> o_interp_head;
-    FieldOffset<GCRuntimeState> o_gc;
+    FieldOffset<char> o_gc;
```

Comment:
Hummm why `char`? Isn't this a pointer?

Reply:
Yeah, but it's an inline structure whose layout we don't know at compile time, and so there's not a useful version-independent type we can use to refer to it by here. `char` is a hack to deal with the fact that we can't actually dereference this field into a meaningful `GCRuntimeState` directly, and instead need to take a pointer to it and create a `Structure` from that pointer.

I'll add a comment explaining this.